### PR TITLE
ref(actor): `get_actor_identifier` -> `identifier` property

### PR DESCRIPTION
**Pull request by evanpurkhiser**
This is a bit more ergonomic

### CODE REVIEW

File: src/sentry/models/actor.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def get_actor_tuple(self) -> ActorTuple:
     def get_actor_identifier(self):
```
Comment (JoshFerge) [inline src/sentry/models/actor.py:170]: why do you need to keep this function around?

Comment (evanpurkhiser) [reply src/sentry/models/actor.py:170]: This one is still being used. This is on the `Actor` model, I've changed the one on the `ActorTupl`

**Review by JoshFerge — APPROVED** [review]

**Comment by evanpurkhiser** [comment]
Damn some of the places this was changed were not tuples 😢

**Comment by evanpurkhiser** [comment]
All fixed up now

**Pull request merged**